### ISSUE TITLE
nat_inside/outside don't seem to be available as attributes

### ISSUE THREAD

**Issue by FragmentedPacket** [nautobot/pynautobot#38]
```python
In [11]: ip = nb.ipam.ip_addresses.get(address="192.168.1.10/24")

In [12]: ip.nat_inside
Out[12]:

In [13]: ip.serialize()
Out[13]:
{'id': '6845e644-6031-4592-aa4e-1f62a317462c',
 'url': 'http://localhost:8080/api/ipam/ip-addresses/6845e644-6031-4592-aa4e-1f62a317462c/',
 'family': 4,
 'address': '192.168.1.10/24',
 'vrf': None,
 'tenant': None,
 'status': 'active',
 'role': None,
 'assigned_object_type': None,
 'assigned_object_id': None,
 'assigned_object': None,
 'nat_inside': '2cbff972-943f-4862-b9fc-b943ee2066fc',
 'nat_outside': None,
 'dns_name': '',
 'description': '',
 'tags': [],
 'custom_fields': {},
 'created': '2022-04-03',
 'last_updated': '2022-04-17T14:37:25.641707Z',
 'display': '192.168.1.10/24'}
```

It seems to correlate other FK fields such as tenant.

**Comment by FragmentedPacket** [comment]
Closed via #41

**Comment by jvanderaa** [comment]
In reality, IPv6 should be the answer here. @FragmentedPacket  :)
